.openpublishing.redirection.json

```diff
@@ -294,6 +294,10 @@
         "redirect_url": "/visualstudio/ide/visual-studio-ide"
     },
     {
+        "source_path": "docs/visual-basic/getting-started/breaking-changes-in-visual-studio-2015.md", 
```

Comment:
I thought this was like a landing doc for all of the versions: https://msdn.microsoft.com/en-us/library/cc714070.aspx

Will that continue to version forward with future releases (i.e., will it continue to be cumulative over time)?